src/components/error-message/index.md.njk

```diff
@@ -21,7 +21,9 @@ Use standard messages for different components.
 
 ## When not to use this component
 
-Do not use error messages to tell users that they are not eligible or do not have permission to do something. Instead, take them to a page which tells them they’re not eligible and gives them useful information about what to do next.
+Do not use error messages to tell a user that they are not eligible or do not have permission to do something. Or to tell them about a lack of capacity or other problem the user cannot fix - because the problem is with the service rather than with the information the user has provided.
+
+Instead, take them to a page which explains the problem (for example, telling them why they’re not eligible) and provides useful information about what to do next.
```

Comment:
```suggestion
Instead, take the user to a page that explains the problem (for example, telling them why they’re not eligible) and provides useful information about what to do next.
```

I'm never 100% sure but... I think it should be 'that' rather than 'which'? Or are they interchangeable here?